src/20250320_Multi_RiskStewardParameterUpdatesPhase3/RiskStewardParameterUpdatesPhase3_20250320.s.sol

```diff
@@ -0,0 +1,447 @@
+    actionsPolygon[0] = GovV3Helpers.buildAction(
+      type(AaveV3Polygon_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[1] = GovV3Helpers.buildPolygonPayload(vm, actionsPolygon);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsAvalanche = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsAvalanche[0] = GovV3Helpers.buildAction(
+      type(AaveV3Avalanche_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[2] = GovV3Helpers.buildAvalanchePayload(vm, actionsAvalanche);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsOptimism = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsOptimism[0] = GovV3Helpers.buildAction(
+      type(AaveV3Optimism_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[3] = GovV3Helpers.buildOptimismPayload(vm, actionsOptimism);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsArbitrum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsArbitrum[0] = GovV3Helpers.buildAction(
+      type(AaveV3Arbitrum_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[4] = GovV3Helpers.buildArbitrumPayload(vm, actionsArbitrum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsMetis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsMetis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Metis_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[5] = GovV3Helpers.buildMetisPayload(vm, actionsMetis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsGnosis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsGnosis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Gnosis_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[6] = GovV3Helpers.buildGnosisPayload(vm, actionsGnosis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBase = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBase[0] = GovV3Helpers.buildAction(
+      type(AaveV3Base_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[7] = GovV3Helpers.buildBasePayload(vm, actionsBase);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsScroll = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsScroll[0] = GovV3Helpers.buildAction(
+      type(AaveV3Scroll_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[8] = GovV3Helpers.buildScrollPayload(vm, actionsScroll);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBNB = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBNB[0] = GovV3Helpers.buildAction(
+      type(AaveV3BNB_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[9] = GovV3Helpers.buildBNBPayload(vm, actionsBNB);
+
+    IPayloadsControllerCore.ExecutionAction[]
```

Comment:
For zksync payload needs to be deployed and registered first, then add payloadId etc here.

example: https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20250228_Multi_ClinicStewardActivation/ClinicStewardActivation_20250228.s.sol#L387-L394